dublin-guide.md

```diff
@@ -0,0 +1,3 @@
+## Places to eat in Dublin
+
+# The White Moose Cafe
```

Comment:
Full Irish breakfast. And be nice to the owner, he's vindictive, but entertaining, on his Facebook page.